### ISSUE TITLE
Write "Testing" lesson plan

### ISSUE THREAD

**Issue by k4y3ff** [Techtonica/curriculum#86]
The lesson plan should live at [`testing/testing.md`](https://github.com/Techtonica/curriculum/blob/master/testing/testing.md).

Acceptance Criteria:
- [ ] The lesson has a defined time scope (45 minutes? 2 hours? etc.)
- [ ] There are clear objectives
- [ ] The activity helps different kinds of learners (e.g. spatial, auditory, verbal, kinesthetic)
- [ ] There is a cheatsheet or other handout
- [ ] There is an assessment

Note that [self and peer assessments are preferable to grades](https://www.mindsetkit.org/topics/assessments-growth-mindset-math/using-self-peer-feedback-assessments-learning).

**Comment by vegetabill** [comment]
still working on this as of tonight

**Comment by MichelleGlauser** [comment]
It is now here: https://github.com/Techtonica/curriculum/blob/master/testing-and-tdd/testing-and-tdd.md

**Comment by Karla-Isabel-Sandoval** [comment]
@vegetabill Thanks for working on the TDD Topic Outline back in August 2017! Would you be interested in creating a video, meaning, using your voice to create a walkthrough of the slides, https://drive.google.com/open?id=1viDJyLHXVs-VioUEatvWHDZeOCCmPxZNPHzH0C393rg, using something like QuickTime?

**Comment by vegetabill** [comment]
I can give it a shot. When would you need it by?

**Comment by Karla-Isabel-Sandoval** [comment]
@vegetabill Thank You for giving it a shot! We would need it by March 17, 2018. Would that be a do-able time frame for you?

**Comment by vegetabill** [comment]
Yes I think I can do that. I'll let you know if I have questions or have a draft that I can get some feedback on.

**Comment by vegetabill** [comment]
I was planning on working on it this weekend but unfortunately I've been sick so actually I don't think I'll be able to get this done. Probably it would be better for someone else to work on it. Sorry!

**Comment by Karla-Isabel-Sandoval** [comment]
@vegetabill hope you feel better soon. Thanks for the update. Will look for another volunteer to work on it.
